README.md

```
@@ -16,9 +16,9 @@
 
 | Articles Infinite Scrolling | Bookmark Article |
 | :---: | :---: |
-| <img src="docs/articles_infinite_scrolling_demo.gif" width="250"> | <img src="docs/bookmark_article_demo.gif" width="250"> |
+| <img src="docs/media/articles_infinite_scrolling_demo.gif" width="250"> | <img src="docs/media/bookmark_article_demo.gif" width="250"> |
```

markdownlint-cli2 (0.23.0)

[warning] Images should have alternate text (alt text)

(MD045, no-alt-text)